src/ui/locales/de.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Zuordnung umschalten",
+    "AttributionControl.MapFeedback": "Karten-Feedback",
+    "FullscreenControl.Enter": "Vollbildmodus aktivieren",
+    "FullscreenControl.Exit": "Vollbild beenden",
+    "GeolocateControl.FindMyLocation": "Finden Sie meinen Standort",
+    "GeolocateControl.LocationNotAvailable": "Ort nicht verfügbar",
+    "LogoControl.Title": "Mapbox-Logo",
+    "Map.Title": "Karte",
+    "NavigationControl.ResetBearing": "Setzen Sie die Peilung auf Nord zurück",
```

Comment:
```suggestion
    "NavigationControl.ResetBearing": "Setzen Sie die Peilung auf Norden zurück",
```

Reply:
Same as above, I think this is a human giving instructions to a machine, so, aggregating the previous comments, I suggest

``` suggestion
    "NavigationControl.ResetBearing": "Setze die Orientierung auf Norden zurück",
```